setup.sh

```diff
@@ -47,11 +55,31 @@ else
 fi
 
 # get CMSSW release
-export SCRAM_ARCH=slc6_amd64_gcc630
+if [[ `uname -r` == *"el6"* ]]; then
+    SLC_VERSION="slc6"
+elif [[ `uname -r` == *"el7"* ]]; then
+    SLC_VERSION="slc7"
+else
+    echo "WARNING::Unknown SLC version. Defaulting to SLC7."
```

Comment:
default should be sl6 as long as that is the CMS default

Reply:
Fixed